libs/test/SmartSyncTest/src/com/salesforce/androidsdk/smartsync/manager/ManagerTestCase.java

```diff
@@ -131,5 +143,70 @@ private RestClient initRestClient() throws Exception {
                 null, null, null, null, null, null);
         return new RestClient(clientInfo, authToken, httpAccess, null);
     }
-    
+
+    /**
+     * Helper methods to create "count" of test records
+     * @param count
+     * @return map of id to name for the created accounts
+     * @throws Exception
+     */
+    public Map<String, String> createRecordsOnServer(int count, String objectType) throws Exception {
+        Map<String, String> idToValues = new HashMap<String, String>();
+        for (int i = 0; i < count; i++) {
+
+            // Request.
+            String fieldValue = "";
+            Map<String, Object> fields = new HashMap<String, Object>();
+            //add more object type if need to support to use this API
+            //to create a new record on server
+            switch (objectType) {
+                case Constants.ACCOUNT:
+                    fieldValue = createRecordName(objectType);
+                    fields.put(Constants.NAME, fieldValue);
+                    break;
+                case Constants.OPPORTUNITY:
+                    fieldValue = createRecordName(objectType);
+                    fields.put(Constants.NAME, fieldValue);
+                    fields.put("StageName", "Prospecting");
+                    DateFormat formatter = new SimpleDateFormat("yyyy-MM-dd");
+                    fields.put("CloseDate", formatter.format(new Date()));
+                    break;
+                default:
+                    break;
+
+            }
+
+            RestRequest request = RestRequest.getRequestForCreate(ApiVersionStrings.getVersionNumber(targetContext), objectType, fields);
+
+            // Response.
+            RestResponse response = restClient.sendSync(request);
```

Comment:
Maybe add a couple of ```assert``` statements after the request is made:
* ```assertNotNull("Response should not be null", response)```;
* ```assertTrue("Response status should be success", response.isSuccess());```

We don't want the unit test to crash on network errors or such.